### DISCUSSION TITLE
high-level changelog/progress missing

### DISCUSSION THREAD

**Discussion by laf0rge** [lethalbit/squishy#40] [Q&A] [answered]
I'm very excited about this project, and it's great how many resources (doucmentation, HDL, board layout files, etc.) you have released even at such an early stage of the project.

However, what is a bit missing for me as an outsider:  What were the problems / restraints of rev1 hardware? What were the reasons to move to rev2? What are the  key high-level differntiators?

Those kind of questions (or rather, the answers thereof) would be important in order to judge if it makes any sense to at all to build a rev1 (rather than waiting for rev2) for somebody who wants to tinker with the project in its early stages.

Thanks for considering to add related information somewhere in some form.  Or point me to it, if I have missed it.

Happy hacking!

**Comment by lethalbit** [comment, marked as answer]
Thank you for your interest! 

The main issues with rev1 over rev2 are as follows:
 * rev1 is SCSI-1 HVD only
 * There is an issue with some of the values for the pull-ups/pull-downs on some of the enable lines for things like `TERMPWR`
 * The termination itself is always-on when providing `TERMPWR`.

Those are the big ones that I can recall off the top of my head, and in general I'm not as happy as I could be with the rev1 hardware in general so i'm hesitant for people build it / rely on it. rev2 aims to change that with it being better designed and thought out, as well as more flexible, The downside is that as a result it's substantially more complex, so that along with some things going on in life it's been rough going, but with things settling down I hope to get things moving proper again with getting rev2 up and going.

I hope that cleared things up, and if you have any other questions feel free to ask!

**Comment by lethalbit** [comment]
As this is still open, I would also like to raise a hardware errata I found while doing some review, it turns out the the `~OE` on the level shifters has been mistakenly held high. The fix is easy enough, just cut the trace running to pin 9 and bridge it with pin 8 for U2, U8, U11, U16, U20, U23, U27, U30, and U33.


Ooops, this wasn't found until now due to me being unable to fully test the rev1 hardware at all. But much more diligence is going into rev2 so. I hope you can wait for that @laf0rge

**Comment by laf0rge** [comment]
> But much more diligence is going into rev2 so. I hope you can wait for that @laf0rge

Well, any user will either have to wiat as long as it takes, or find time to help speed up the project themselves.  You convinced me that it's not useful to build a rev1, so I'll wait.  There's many other pet projects with desparate cries for attention to work on meanwhile.

btw: My initial use case would have been in the context of analyzing what is special about the SCSI drives that are used in Siemens EWSD (a series of 1980s/90s digital telephone exchanges of which a group I'm part of has recently "rescued" one to keep this retronetworking tech alive.   We currently have working original drives, but it's interesting to know  what exactly is making the software in the exchange work only with original drives and not others.  It might be as simple as just cloning the INQUIRY data, but there are also rumours that the drives actually have custom firmware on them.  Having a SCSI bus tracer as long as we still have working HDD would enable us to clarify if there's anything unusual happening on the bus.  Eventually the drives should be replaced by drive simulators.

https://discourse.osmocom.org/t/die-rettung-der-ewsd/30 provides some background (including pictures from the scsi disk dumping and disassembly of that exchange) in case you're curious.
